update-and-commit-translations.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Script to pull transifex translation files, and commit them to git if there are any changes.
+
+tx pull -f --mode=reviewed
+
+if [[ -z $(git diff-index --quiet HEAD --) ]]; then
```

Comment:
@cintiadr I know nothing at all about writing shell scripts. So, I will make this change, but if I get it wrong again feel free to directly edit in the PR.